gamestonk_terminal/comparison_analysis/ca_controller.py

```diff
@@ -318,7 +318,7 @@ def call_technical(self, other_args: List[str]):
 
     def call_po(self, _):
         """Open Portfolio Optimization menu with ticker and similar"""
-        port_optimization_controller.menu([self.ticker] + self.similar)
+        return po_controller.menu([self.ticker] + self.similar)
```

Comment:
you can write a comment above that it's to either leave PO terminal to go back to main menu, or to quit. It doesn't go back to CA menu

Reply:
Will do.  Ill also add an option to go back to the ca option